src/components/Booklist.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import AddNewBook from './AddNewBook';
+
+const Booklist = () => (
+  <div>
+    <AddNewBook />
+    <h1>Book List</h1>
+  </div>
+);
```

Comment:
- [ ] The `BookList` component should contain `Book` as a child component, kindly update your `BookList` component and make sure to pass each book data as `props`.